cg/meta/workflow/mutant.py

```diff
@@ -108,12 +113,15 @@ def get_sample_parameters(self, sample_obj: models.Sample) -> MutantSampleConfig
 
     def get_internal_negative_controls(self, case_id: str) -> List[models.Sample]:
         """Retrieve a list of lims ids for negative controls created by Clinical Genomics"""
+
         case_obj = self.status_db.family(case_id)
         samples: List[models.Sample] = [link.sample for link in case_obj.links]
-        ticket_id: int = samples[0].ticket_number
-        # 1. ticket_id will lead to samples that all come from the same pool
-        # 2. In the pool from "1." there will be samples from another ticket than ticket_id given as input
-        # 3. Return a list of sample objects from "2."
+        sample: models.Sample = samples[0]
+        pools = lims.get_artifacts(samplelimsid=sample.internal_id, type="Analyte",
+                                   process_type=MASTER_STEPS_UDFS['cov_pooling_step'])
+        pool: Artifact = LimsAPI.get_latest_artifact(lims_artifacts=pools)
+        negative_controls: List[Sample] = LimsAPI.get_controls(lims_artifact=pool)
+        return [self.status_db.sample(lims_sample.id) for lims_sample in negative_controls]
```

Comment:
We should be clear that this is about internal negative controls, not any negative control. There will be negative controls in that pool that are from the customer that will contain reads since it is processed differently in the lab.